index.html

```diff
@@ -47,6 +47,7 @@
     <script type="text/javascript" src="js/plugins.js"></script>
     <script type="text/javascript" src="js/imgur.js"></script>
     <script type="text/javascript" src="3rdparty/favico-0.3.10.min.js"></script>
+    <script type="text/javascript" src="cordova.js"></script>
```

Comment:
Might be a good idea to create a Makefile in the cordova repository then that does the necessary changes before building?